test/e2e/utils/record-helpers.js

```diff
@@ -239,7 +239,34 @@ exports.testPresentation = function (tableParams) {
         })
     });
 
-    it("visible column related table with inline inbound fk should display 'None' in markdown disply mode if no data was found.",function(){
+    it("click event on image_id in inline display should open new tab with file details",function(){
+        var allHandle;
+        chaisePage.waitForElement(element(by.id('entity-booking'))).then(function(){
+           return browser.executeScript("return $('#divRecordSet > table > tbody > tr:nth-child(1) > td:nth-child(5) > div > div > span > a')");
```

Comment:
Add a comment saying what element this selector is selecting. 

A better alternative would be to add specific ID's to elements or general classes so you don't have to crawl through the html structure. If we change the html structure, it will break this test case so it might be better to identify the element rather than crawling the HTML.

Reply:
I remember pointing this way back last year that we should not rely upon nested html for capturing elements. But was told that it's what is expected so that even if a slight change happens in the DOM it should fail and we get to know the change.
Nonetheless, I tweaked it to reduce to just 3 levels and added a comment.